nixos/modules/flyingcircus/packages/fcmaintenance/setup.py

```diff
@@ -52,6 +52,7 @@ def run_tests(self):
     cmdclass={'test': PyTest},
     extras_require={
         'dev': test_req + ['pytest-cov'],
```

Comment:
I'd rather use "dev" for everything. Having two variants is a bit confusing, isn't it?